core/src/avm2/globals/flash/media/soundchannel.rs

```diff
@@ -88,15 +88,14 @@ pub fn sound_transform<'gc>(
     this: Option<Object<'gc>>,
     _args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
-    if let Some(instance) = this
-        .and_then(|this| this.as_sound_channel())
-        .and_then(|channel| channel.instance())
-    {
-        let dobj_st = activation.context.local_sound_transform(instance).cloned();
+    if let Some(channel) = this.and_then(|this| this.as_sound_channel()) {
+        let dobj_st = channel
+            .instance()
+            .and_then(|instance| activation.context.local_sound_transform(instance))
+            .cloned()
+            .unwrap_or_default();
```

Comment:
unfortunately this is a bigger issue, looks like the transform is somehow bound to the channel, not to the instance?
```
	        var a = channel.soundTransform;
	        a.volume = 0.5;
	        channel.soundTransform = a;
	        var b = channel.soundTransform;
	        trace(a == b); // false, so the getter always returns a copy
	        trace(b.volume); // 0.5
```